src/chigame/games/views.py

```diff
@@ -209,3 +211,42 @@ def search_results(request):
     context = {"query_type": "Games", "object_list": object_list}
 
     return render(request, "pages/search_results.html", context)
+
+
+# Placeholder Game
+@login_required
+def coin_flip_game(request, pk):
+    return render(request, "games/game_coinflip.html", {"lobby_id": pk})
+
+
+@login_required
+def check_guess(request, pk):
+    user_guess = request.POST.get("user_guess")
+    coin_result = choice(["heads", "tails"])
+    correct_guess = user_guess == coin_result
+
+    lobby = get_object_or_404(Lobby, id=pk)
+
+    # Create Match instance linked to the fetched Lobby
+    match = Match.objects.create(
```

Comment:
I'm getting an error here because I already have a match for this particular lobby. 
<img width="782" alt="image" src="https://github.com/uchicago-cs/chigame/assets/102563648/18549f0a-b084-47c0-9884-3918e2b1512b">
I know this is just a dummy game, but I'm not sure if building on top of a pre-existing lobby is the best option because of this possibility. An alternative could be to check for match instances first then prevent this page from being reached if there is one.